tests/test_source.py

```diff
@@ -0,0 +1,119 @@
+# pylint: disable=redefined-outer-name
+import typing as tp
+
+import attr
+import mock
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)